### ISSUE TITLE
Dashboard Widget is Displayed for Users With the Subscriber Role

### ISSUE THREAD

**Issue by TheJeffr0** [CastosHQ/Seriously-Simple-Stats#65]
I discovered today during testing that while logged in as a user with the Subscriber role which is the default role for new users, the Stats dashboard widget is displayed to them. In my opinion, this is sort of a data leak as I don't think users with the Subscribers role should be privy to that kind of data. Suggested solution us to make it so that the Stats Dashboard Widget is only displayed to those with the Administrator role. @jonathanbossenger

**Comment by jonathanbossenger** [comment]
@TheJeffr0 that's a valid point, thanks for bringing this to our attention. I will schedule this into our next plugin update cycle.

In the meantime, if you feel comfortable editing the stats plugin code, you can wrap the add_meta_box call which creates the stats widget in current_user_can condition.

So in line 1161 of /php/classes/class-ssp-stats.php change the add_dashboard_widget method to the following:

```
if ( current_user_can( 'manage_podcast' ) ) {
	add_meta_box( 'ssp_stats_dashboard_widget', __('Seriously Simple Stats - Overview', 'seriously-simple-stats' ), array( $this, 'dashboard_widget_callback' ), 'dashboard', 'normal', 'high' );
}
```

**Comment by jonathanbossenger** [comment]
@TheJeffr0 this has been fixed in the 1.2.9 release, thanks for your patience.
